scripts/website/expand_templates/main.go

```diff
@@ -40,7 +41,7 @@ func rewriteDocs(replacements map[string]string) error {
 			return processDoc(path, replacements, madeReplacements)
 		})
 	if err != nil {
-		return fmt.Errorf("failed to walk dir: %w", err)
+		return fmt.Errorf("walk dir: %w", err)
```

Comment:
There are a lot of places where we can remove the string "failed to " from an error message: https://github.com/search?q=repo%3Agolangci%2Fgolangci-lint+%22fmt.Errorf%28%5C%22failed+to+%22&type=code
However, this should be done as a separate refactoring PR.

Reply:
It was an element of the review but I don't want to change all the error messages (please don't create a PR for that).